### PR TITLE
No footer list underline

### PR DESCRIPTION
**Pull request by jordesign**
re #158 

This removes the underline on links within li elements in the footer widgets.
It also offers an alternative style of abbr so as not to clash with the horizontal rules throughout the site.

image

user: @jordesign

### CODE REVIEW

**Comment by melchoyce** [comment]
Hey @jordesign, would you mind splitting these out into two PRs? One for removing the footer list link underlines, and one for the alternate `abbr` styling?

**Comment by jordesign** [comment]
This PR is now just the underline on footer widgets - with new abbr styling removed

File: style.css [hunk 1]
```diff
@@ -847,7 +847,10 @@ a:visited .nav-title,
 .widget .widget-title a,
 .widget .widget-title a:visited,
 .widget ul li a,
-.widget ul li a:visited {
+.widget ul li a:visited, 
+.site-footer .widget-area ul li a,
+.site-footer .widget-area ul li a:visited
+ {
```
Comment (ntwb) [inline style.css]: The opening brace should be on the previous line with a single space preceeding it.

**Review by ntwb — CHANGES_REQUESTED** [review]

**Comment by melchoyce** [comment]
Looks good 👍

**Pull request merged**